src/main/java/com/khoros/twitterapp/services/CacheUp.java

```diff
@@ -8,14 +8,23 @@
 
 public class CacheUp {
     // clean up interval 1 week
-    private Set<Status> cacheSet = new HashSet<>();
+    private Set<Status> homeTimelineSet = new HashSet<>();
```

Comment:
this does work.  Come up with a single cache implementation that can be reused for different items without having to add additional data types/methods